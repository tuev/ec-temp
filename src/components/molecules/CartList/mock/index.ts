import mockImage from './img.png'
import { CartItemProps } from '../CartList.types'

const cartItems: CartItemProps[] = [
  {
    image: mockImage,
    size: 'M',
    quantity: 1,
    name: 'Product name 1',
    id: '123123',
    price: 32,
  },
  {
    image: mockImage,
    size: 'L',
    quantity: 2,
    name: 'Product name 2',
    id: '123122',
    price: 32,
  },
  {
    image: mockImage,
    size: 'XL',
    quantity: 3,
    name: 'Product name 3',
    id: '123124',
    price: 32,
  },
]

export default cartItems
