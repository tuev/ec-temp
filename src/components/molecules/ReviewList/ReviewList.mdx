# Stories mdx for ReviewList

import { Meta, Story } from '@storybook/addon-docs/blocks'
import * as stories from './ReviewList.stories.tsx'
import ReviewList from './ReviewList.tsx'

<Story name="basic" />