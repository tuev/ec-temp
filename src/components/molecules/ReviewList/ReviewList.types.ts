import { SpaceProps } from 'styled-system'
import { IReviewItem } from 'components/molecules/ReviewItem/ReviewItem.types'

export interface IReviewList {
  list?: IReviewItem[]
}

export type BaseReviewRowProps = SpaceProps
