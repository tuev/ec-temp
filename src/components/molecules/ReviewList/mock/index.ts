export const reviewList = [
  {
    id: 1,
    rating: 3,
    fullname: 'Pablo Permins',
    dateCreated: 'an hour ago',
    reviewText:
      'Got this through the post the other day and right from opening the packet I knew this was quality, put it on and I was right!! Well done',
    likeCount: 0,
    commentCount: 20,
  },
]
