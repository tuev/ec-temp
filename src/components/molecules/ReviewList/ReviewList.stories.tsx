/**
 *
 *  Stories for ReviewList
 *
 */

import React from 'react'
import ReviewList from './ReviewList'
import { reviewList } from './mock'

export default {
  component: ReviewList,
  includeStories: ['basic'],
  title: 'ReviewList',
}

export const basic = () => (
  <div>
    <ReviewList list={reviewList} />
  </div>
)
