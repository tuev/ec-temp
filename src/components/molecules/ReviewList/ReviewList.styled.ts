/**
 *
 * Styled for ReviewList
 *
 */

import styled from 'styled-components'
import { space } from 'styled-system'
import { BaseReviewRowProps } from './ReviewList.types'
import { Grid } from '@material-ui/core'
import { Typography } from '../../atoms'
import { COLOR } from '../../../theme/colors'

export const ReviewListRow = styled(Grid)<BaseReviewRowProps>`
  ${space}
`

export const ReviewListWrapper: React.FC = styled.div`
  ${space}
`
export const Warning = styled(Typography)`
  color: ${COLOR.black2};
`
