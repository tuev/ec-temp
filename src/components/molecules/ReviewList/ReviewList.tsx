/**
 *
 * ReviewList
 *
 */

import React, { FC } from 'react'
import { Title, Text } from '../ReviewForm/ReviewForm.styled'
import { Warning } from './ReviewList.styled'
import { Grid } from '@material-ui/core'
import { Rating, Typography, BlogIcon } from 'components/atoms'
import { ReviewListWrapper, ReviewListRow } from './ReviewList.styled'
import { BaseReviewListProps } from './ReviewList.types'

const ReviewList: FC<BaseReviewListProps> = (props: BaseReviewListProps) => {
  const { list } = props
  return (
    <ReviewListWrapper>
      <Title customvariant="header2" mb={4}>
        Product reviews
      </Title>
      {list &&
        list.map(review => {
          const {
            id,
            rating,
            fullname,
            dateCreated,
            reviewText,
            likeCount,
            commentCount,
          } = review
          return (
            <ReviewListRow container key={id} mb={4}>
              <Grid item xs={12} md={3}>
                <Rating value={rating} readOnly mb={4} />
                <Text customvariant="body1">{fullname}</Text>
                <Warning customvariant="body2">{dateCreated}</Warning>
              </Grid>
              <Grid item xs={12} md={9}>
                <Typography customvariant="body1" mb={3}>
                  {reviewText}
                </Typography>
                <Grid container>
                  <Grid item xs={2}>
                    <BlogIcon icon="like" count={likeCount} />
                  </Grid>
                  <Grid item xs={2}>
                    <BlogIcon icon="comment" count={commentCount} />
                  </Grid>
                </Grid>
              </Grid>
            </ReviewListRow>
          )
        })}
    </ReviewListWrapper>
  )
}

export default ReviewList
