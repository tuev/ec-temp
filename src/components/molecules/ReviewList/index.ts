/**
 *
 * Index for ReviewList
 *
 */

import ReviewList from './ReviewList'

export default ReviewList
