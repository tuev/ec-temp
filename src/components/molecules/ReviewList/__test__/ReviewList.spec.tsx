/**
 *
 * Tests for ReviewList
 *
 */

import React from 'react'

import { render, fireEvent } from '@testing-library/react'
import ReviewList from '..'

import { reviewList } from '../mock'

describe('App ReviewList', () => {
  it('Test get default ReviewList no params ', () => {
    // test snapshot
    const wrapper = render(<ReviewList />)
    expect(wrapper).toMatchSnapshot()
  })

  it('Test get default ReviewList with list ', () => {
    // test snapshot
    const wrapper = render(<ReviewList list={reviewList} />)
    expect(wrapper).toMatchSnapshot()
  })
})
