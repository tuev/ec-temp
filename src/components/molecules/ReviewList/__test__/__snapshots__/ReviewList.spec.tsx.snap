// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App ReviewList Test get default ReviewList no params  1`] = `
Object {
  "asFragment": [Function],
  "baseElement": .c0 {
  margin-bottom: 32px;
  font-size: 24px;
  line-height: 1.5;
  font-family: 'Montserrat Alternates';
  font-weight: bold;
  color: #2E2E2E;
}

<body>
    <div>
      <div
        class=""
      >
        <p
          class="MuiTypography-root c0 MuiTypography-body1"
          customvariant="header2"
          mb="4"
        >
          Product reviews
        </p>
        <div>
          No reviews
        </div>
      </div>
    </div>
  </body>,
  "container": <div>
    <div
      class="sc-fzplWN JlMvU"
    >
      <p
        class="MuiTypography-root sc-AxjAm jgyCZp sc-AxirZ ktnnZK MuiTypography-body1"
        customvariant="header2"
        mb="4"
      >
        Product reviews
      </p>
      <div>
        No reviews
      </div>
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;

exports[`App ReviewList Test get default ReviewList with list  1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    .c0 {
  margin-bottom: 32px;
  font-size: 24px;
  line-height: 1.5;
  font-family: 'Montserrat Alternates';
  font-weight: bold;
  color: #2E2E2E;
}

.c3 {
  font-size: 16px;
  line-height: 1.5;
  font-family: 'Open Sans';
  color: #2E2E2E;
}

.c5 {
  font-size: 12px;
  line-height: 2;
  font-family: 'Open Sans';
  color: #2E2E2E;
}

.c7 {
  margin-bottom: 16px;
  font-size: 16px;
  line-height: 1.5;
  font-family: 'Open Sans';
  color: #2E2E2E;
}

.c4 {
  font-weight: 600;
}

.c8 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  -webkit-box-pack: space-around;
  -webkit-justify-content: space-around;
  -ms-flex-pack: space-around;
  justify-content: space-around;
  max-width: 80px;
  cursor: pointer;
}

.c9 {
  width: 32px;
  height: 32px;
  border-radius: 50%;
  background-color: #F2F2F2;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
}

.c2 {
  margin-bottom: 32px;
}

.c1 {
  margin-bottom: 32px;
}

.c6 {
  color: #B4B4B4;
}

<div>
      <div
        class=""
      >
        <p
          class="MuiTypography-root c0 MuiTypography-body1"
          customvariant="header2"
          mb="4"
        >
          Product reviews
        </p>
        <div
          class="MuiGrid-root c1 MuiGrid-container"
          mb="4"
        >
          <div
            class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12 MuiGrid-grid-md-3"
          >
            <span
              aria-label="3 Stars"
              class="MuiRating-root c2 MuiRating-readOnly"
              mb="4"
              role="img"
            >
              <span
                class="MuiRating-icon MuiRating-iconFilled"
              >
                <svg
                  aria-hidden="true"
                  class="MuiSvgIcon-root MuiSvgIcon-fontSizeInherit"
                  focusable="false"
                  role="presentation"
                  viewBox="0 0 24 24"
                >
                  <path
                    d="M12 17.27L18.18 21l-1.64-7.03L22 9.24l-7.19-.61L12 2 9.19 8.63 2 9.24l5.46 4.73L5.82 21z"
                  />
                </svg>
              </span>
              <span
                class="MuiRating-icon MuiRating-iconFilled"
              >
                <svg
                  aria-hidden="true"
                  class="MuiSvgIcon-root MuiSvgIcon-fontSizeInherit"
                  focusable="false"
                  role="presentation"
                  viewBox="0 0 24 24"
                >
                  <path
                    d="M12 17.27L18.18 21l-1.64-7.03L22 9.24l-7.19-.61L12 2 9.19 8.63 2 9.24l5.46 4.73L5.82 21z"
                  />
                </svg>
              </span>
              <span
                class="MuiRating-icon MuiRating-iconFilled"
              >
                <svg
                  aria-hidden="true"
                  class="MuiSvgIcon-root MuiSvgIcon-fontSizeInherit"
                  focusable="false"
                  role="presentation"
                  viewBox="0 0 24 24"
                >
                  <path
                    d="M12 17.27L18.18 21l-1.64-7.03L22 9.24l-7.19-.61L12 2 9.19 8.63 2 9.24l5.46 4.73L5.82 21z"
                  />
                </svg>
              </span>
              <span
                class="MuiRating-icon MuiRating-iconEmpty"
              >
                <svg
                  aria-hidden="true"
                  class="MuiSvgIcon-root MuiSvgIcon-fontSizeInherit"
                  focusable="false"
                  role="presentation"
                  viewBox="0 0 24 24"
                >
                  <path
                    d="M12 17.27L18.18 21l-1.64-7.03L22 9.24l-7.19-.61L12 2 9.19 8.63 2 9.24l5.46 4.73L5.82 21z"
                  />
                </svg>
              </span>
              <span
                class="MuiRating-icon MuiRating-iconEmpty"
              >
                <svg
                  aria-hidden="true"
                  class="MuiSvgIcon-root MuiSvgIcon-fontSizeInherit"
                  focusable="false"
                  role="presentation"
                  viewBox="0 0 24 24"
                >
                  <path
                    d="M12 17.27L18.18 21l-1.64-7.03L22 9.24l-7.19-.61L12 2 9.19 8.63 2 9.24l5.46 4.73L5.82 21z"
                  />
                </svg>
              </span>
            </span>
            <p
              class="MuiTypography-root c3 c4 MuiTypography-body1"
              customvariant="body1"
            >
              Pablo Permins
            </p>
            <p
              class="MuiTypography-root c5 c6 MuiTypography-body1"
              customvariant="body2"
            >
              an hour ago
            </p>
          </div>
          <div
            class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12 MuiGrid-grid-md-9"
          >
            <p
              class="MuiTypography-root c7 MuiTypography-body1"
              customvariant="body1"
              mb="3"
            >
              Got this through the post the other day and right from opening the packet I knew this was quality, put it on and I was right!! Well done
            </p>
            <div
              class="MuiGrid-root MuiGrid-container"
            >
              <div
                class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-2"
              >
                <div
                  class="c8"
                >
                  <div
                    class="c9"
                  >
                    <svg
                      aria-hidden="true"
                      class="MuiSvgIcon-root MuiSvgIcon-colorError MuiSvgIcon-fontSizeSmall"
                      focusable="false"
                      role="presentation"
                      viewBox="0 0 24 24"
                    >
                      <path
                        d="M16.5 3c-1.74 0-3.41.81-4.5 2.09C10.91 3.81 9.24 3 7.5 3 4.42 3 2 5.42 2 8.5c0 3.78 3.4 6.86 8.55 11.54L12 21.35l1.45-1.32C18.6 15.36 22 12.28 22 8.5 22 5.42 19.58 3 16.5 3zm-4.4 15.55l-.1.1-.1-.1C7.14 14.24 4 11.39 4 8.5 4 6.5 5.5 5 7.5 5c1.54 0 3.04.99 3.57 2.36h1.87C13.46 5.99 14.96 5 16.5 5c2 0 3.5 1.5 3.5 3.5 0 2.89-3.14 5.74-7.9 10.05z"
                      />
                    </svg>
                  </div>
                  <span />
                </div>
              </div>
              <div
                class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-2"
              >
                <div
                  class="c8"
                >
                  <div
                    class="c9"
                  >
                    <svg
                      aria-hidden="true"
                      class="MuiSvgIcon-root MuiSvgIcon-fontSizeSmall"
                      focusable="false"
                      role="presentation"
                      viewBox="0 0 24 24"
                    >
                      <path
                        d="M20 2H4c-1.1 0-2 .9-2 2v18l4-4h14c1.1 0 2-.9 2-2V4c0-1.1-.9-2-2-2zm0 14H6l-2 2V4h16v12z"
                      />
                    </svg>
                  </div>
                  <span>
                    20
                  </span>
                </div>
              </div>
            </div>
          </div>
        </div>
      </div>
    </div>
  </body>,
  "container": <div>
    <div
      class="sc-fzplWN JlMvU"
    >
      <p
        class="MuiTypography-root sc-AxjAm jgyCZp sc-AxirZ ktnnZK MuiTypography-body1"
        customvariant="header2"
        mb="4"
      >
        Product reviews
      </p>
      <div
        class="MuiGrid-root sc-fzpans GsNoN MuiGrid-container"
        mb="4"
      >
        <div
          class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12 MuiGrid-grid-md-3"
        >
          <span
            aria-label="3 Stars"
            class="MuiRating-root sc-fzozJi fnALdj MuiRating-readOnly"
            mb="4"
            role="img"
          >
            <span
              class="MuiRating-icon MuiRating-iconFilled"
            >
              <svg
                aria-hidden="true"
                class="MuiSvgIcon-root MuiSvgIcon-fontSizeInherit"
                focusable="false"
                role="presentation"
                viewBox="0 0 24 24"
              >
                <path
                  d="M12 17.27L18.18 21l-1.64-7.03L22 9.24l-7.19-.61L12 2 9.19 8.63 2 9.24l5.46 4.73L5.82 21z"
                />
              </svg>
            </span>
            <span
              class="MuiRating-icon MuiRating-iconFilled"
            >
              <svg
                aria-hidden="true"
                class="MuiSvgIcon-root MuiSvgIcon-fontSizeInherit"
                focusable="false"
                role="presentation"
                viewBox="0 0 24 24"
              >
                <path
                  d="M12 17.27L18.18 21l-1.64-7.03L22 9.24l-7.19-.61L12 2 9.19 8.63 2 9.24l5.46 4.73L5.82 21z"
                />
              </svg>
            </span>
            <span
              class="MuiRating-icon MuiRating-iconFilled"
            >
              <svg
                aria-hidden="true"
                class="MuiSvgIcon-root MuiSvgIcon-fontSizeInherit"
                focusable="false"
                role="presentation"
                viewBox="0 0 24 24"
              >
                <path
                  d="M12 17.27L18.18 21l-1.64-7.03L22 9.24l-7.19-.61L12 2 9.19 8.63 2 9.24l5.46 4.73L5.82 21z"
                />
              </svg>
            </span>
            <span
              class="MuiRating-icon MuiRating-iconEmpty"
            >
              <svg
                aria-hidden="true"
                class="MuiSvgIcon-root MuiSvgIcon-fontSizeInherit"
                focusable="false"
                role="presentation"
                viewBox="0 0 24 24"
              >
                <path
                  d="M12 17.27L18.18 21l-1.64-7.03L22 9.24l-7.19-.61L12 2 9.19 8.63 2 9.24l5.46 4.73L5.82 21z"
                />
              </svg>
            </span>
            <span
              class="MuiRating-icon MuiRating-iconEmpty"
            >
              <svg
                aria-hidden="true"
                class="MuiSvgIcon-root MuiSvgIcon-fontSizeInherit"
                focusable="false"
                role="presentation"
                viewBox="0 0 24 24"
              >
                <path
                  d="M12 17.27L18.18 21l-1.64-7.03L22 9.24l-7.19-.61L12 2 9.19 8.63 2 9.24l5.46 4.73L5.82 21z"
                />
              </svg>
            </span>
          </span>
          <p
            class="MuiTypography-root sc-AxjAm FxMPz sc-AxiKw cvnEaT MuiTypography-body1"
            customvariant="body1"
          >
            Pablo Permins
          </p>
          <p
            class="MuiTypography-root sc-AxjAm dATcSb sc-fznyAO bLPFwF MuiTypography-body1"
            customvariant="body2"
          >
            an hour ago
          </p>
        </div>
        <div
          class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12 MuiGrid-grid-md-9"
        >
          <p
            class="MuiTypography-root sc-AxjAm gyLNiV MuiTypography-body1"
            customvariant="body1"
            mb="3"
          >
            Got this through the post the other day and right from opening the packet I knew this was quality, put it on and I was right!! Well done
          </p>
          <div
            class="MuiGrid-root MuiGrid-container"
          >
            <div
              class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-2"
            >
              <div
                class="sc-AxhUy gWAFto"
              >
                <div
                  class="sc-AxgMl cTuaLz"
                >
                  <svg
                    aria-hidden="true"
                    class="MuiSvgIcon-root MuiSvgIcon-colorError MuiSvgIcon-fontSizeSmall"
                    focusable="false"
                    role="presentation"
                    viewBox="0 0 24 24"
                  >
                    <path
                      d="M16.5 3c-1.74 0-3.41.81-4.5 2.09C10.91 3.81 9.24 3 7.5 3 4.42 3 2 5.42 2 8.5c0 3.78 3.4 6.86 8.55 11.54L12 21.35l1.45-1.32C18.6 15.36 22 12.28 22 8.5 22 5.42 19.58 3 16.5 3zm-4.4 15.55l-.1.1-.1-.1C7.14 14.24 4 11.39 4 8.5 4 6.5 5.5 5 7.5 5c1.54 0 3.04.99 3.57 2.36h1.87C13.46 5.99 14.96 5 16.5 5c2 0 3.5 1.5 3.5 3.5 0 2.89-3.14 5.74-7.9 10.05z"
                    />
                  </svg>
                </div>
                <span />
              </div>
            </div>
            <div
              class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-2"
            >
              <div
                class="sc-AxhUy gWAFto"
              >
                <div
                  class="sc-AxgMl cTuaLz"
                >
                  <svg
                    aria-hidden="true"
                    class="MuiSvgIcon-root MuiSvgIcon-fontSizeSmall"
                    focusable="false"
                    role="presentation"
                    viewBox="0 0 24 24"
                  >
                    <path
                      d="M20 2H4c-1.1 0-2 .9-2 2v18l4-4h14c1.1 0 2-.9 2-2V4c0-1.1-.9-2-2-2zm0 14H6l-2 2V4h16v12z"
                    />
                  </svg>
                </div>
                <span>
                  20
                </span>
              </div>
            </div>
          </div>
        </div>
      </div>
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
